File: 프로그래머스/2/12985. 예상 대진표/예상 대진표.java
import java.util.*;

class Solution
{
    public int solution(int n, int a, int b)
    {
        int answer = 0;
        
        while(n > 0){
            
            a = (int)Math.ceil((double)a/2);
            b = (int)Math.ceil((double)b/2);
            
            if(a == b) break;
            
            n /= 2;
            answer++;
                
        }

        return answer+1;
    }
}

File: 프로그래머스/2/12985. 예상 대진표/예상 대진표.js
function solution(n,a,b)
{
    var answer = 0;

    while(n > 0){
        
        a = Math.ceil(a/2)
        b = Math.ceil(b/2)
        
        if(a==b) break;
        answer++;
            
    }

    return answer+1;
}